packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec

```
@@ -0,0 +1,113 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
+# Generated from foreman_azure_rm-2.0.0.pre1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_azure_rm
+%global plugin_name azure_rm
+%global foreman_min_version FIXME
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.0.0.pre1
+Release: 1%{?foremandist}%{?dist}
+Summary: Azure Resource Manager as a compute resource for The Foreman
+Group: Applications/Systems
+License: GPL-3.0
```

Open in issue in our tooling, but the format gems and RPMs use differs:
```suggestion
License: GPLv3
```

---

updated.